### ISSUE TITLE
Inline instructions

### ISSUE THREAD

**Issue by ignasi35** [lagom/online-auction-java#60]
Add a feature-flag (enabled by default) to display inline instructions on the user's UI. This means adding text snippets in the rendered HTML so that an eager user gets info in the actual UI when using the app.

Having inline instructions should help a new user get a better understanding of what can be done with the app once started. The feature flag will hide the inline instructions and is meant to remove noise if/when demoing the app.

This feature flag will control the data added in #59  (wip #68 ) , #58(done), #57(done) and #56 (done).

The existence of this inline help should be introduced in the README.md.
The steps to disable the feature should be documented in the repo README.md.

**Comment by mehulsr** [comment]
Hi, I have started working on this issue. Please assign it to me @ignasi35 @TimMoore

**Comment by TimMoore** [comment]
Thanks for commenting @mehulsr. We can't assign the issue to someone who is not part of the lagom organization, but with your comment here, hopefully that will prevent other people from trying to work on this. We'll look forward to your pull request! 😄

**Comment by mehulsr** [comment]
Sure, no problem @TimMoore :smile: 
I'll send in the PR soon!

**Comment by ignasi35** [comment]
Hi @mehulsr , did you have a chance to move forward on this issue? Is there anything we can do to unblock your progress?

**Comment by mehulsr** [comment]
Hi @TimMoore 
I worked on this a bit and was then thought I'd wait for the other PR's to get merged. I'll make a PR for the instructions that have been added 😄
